CHANGELOG.md

```diff
@@ -1886,6 +1886,7 @@ Package-specific changes not released in any SDK will be added here just before
 
 ### 📚 3rd party library updates
 
+- Updated `@shopify/flash-list` from `1.7.1` to `1.7.3`
```

Comment:
the changelog looks not correct. you should add it to the top of "Unpublished" section